.github/workflows/build.yaml

```diff
@@ -265,7 +270,7 @@ jobs:
         id: cache-ffmpeg
```

Comment:
Should we cache binaries `~/build` like in `linux-build:`

Reply:
oh, that step can be removed entirely. i don't think it is needed at all. it was just some experiment with artifact path resolution i had to do and forgot to remove it.